examples/test/test_ai_society_example.py

```diff
@@ -26,3 +28,15 @@ def test_ai_society_role_playing_example():
 def test_role_playing_with_function_example():
     with patch('time.sleep', return_value=None):
         examples.function_call.role_playing_with_function.main(ModelType.STUB)
+
+
+def test_role_playing_with_llama2():
```

Comment:
These two tests become meaningless with `ModelType.STUB`.

Reply:
Exactly... But you know I have not yet implemented the functionality to automatically launching server running LLMs as you mentioned. The tests can certainly not be passed.